### ISSUE TITLE
Option not to focus on input mount

### ISSUE THREAD

**Issue by yhranik-vp** [FlowiseAI/FlowiseChatEmbed#181]
Hello,

I'm using a standalone (flowise-fullchatbot) chat widget and, by default, it triggers input focus on component mount.
This results in automatically scrolling to that element on the page, which is unwanted behavior in my case.

I'd love to be able to opt-out of the default input focus.

**Comment by HenryHengZJ** [comment]
```
Chatbot.init({
      chatflowid: '123',
      apiHost: 'http://localhost:3000',
      theme: {
        chatWindow: {
          textInput: {
              autoFocus: false
          },
        }
      }
    });
```

**Comment by HenryHengZJ** [comment]
Released in v1.3.3!
